meteor/package.json

```diff
@@ -109,7 +109,8 @@
 		"vm2": "^3.9.11",
 		"webmidi": "^2.5.3",
 		"winston": "^3.8.2",
-		"xmlbuilder": "^15.1.1"
+		"xmlbuilder": "^15.1.1",
+		"paraphrase": "^3.1.1"
```

Comment:
Looking at the NPM page for this library I'm not sure I quite understand what this library is doing for us that we wouldn't just be able to do through interpolation ourselves.
What is the benefits of adding this library?

Reply:
> do through interpolation ourselves

I'm not sure I understand what you mean. We can't use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation) here, so we need a regex, and a function to resolve the dot notation.
Yes, it is one of those libraries that have less than 50 lines of actual functionality, that you could write yourself, but it is already written and covered by tests